Mathlib/Tactic/ScopedNS.lean

```diff
@@ -18,7 +18,7 @@ open Lean
 `scoped[NS]` is similar to the `scoped` modifier on attributes and notations,
 but it scopes the syntax in the specified namespace instead of the current namespace.
 ```
-scoped[Matrix] infixl:72 " ⬝ᵥ " => Matrix.dotProduct
+scoped[Matrix] infixl:72 " ⬝ᵥ " => dotProduct
 -- declares `*` as a notation for vector dot productss
 -- which is only accessible if you `open Matrix` or `open scoped Matrix`.
```

Comment:
Something like
```suggestion
scoped[Matrix] postfix:1024 "ᵀ" => Matrix.transpose
-- declares `ᵀ` as a notation for matrix transposition
-- which is only accessible if you `open Matrix` or `open scoped Matrix`.
```
(although `transpose` is not defined as `scoped[Matrix] postfix` but merely `scoped postfix` that happens to live in the `Matrix` namespace.)

Reply:
Should it stay `scoped[Matrix] infixl:72 " ⬝ᵥ " => dotProduct` please?

Reply:
Why would you like to keep it the same? My argument for the change is that documentation that shows code in use as an example is more useful since it allows for easy reference with the rest of the library.

Reply:
Ah, sorry, you meant change the comment!